web/src/pages/BookWizard/OverviewPage.tsx

```diff
@@ -31,15 +47,109 @@ export const OverviewPage: React.FC<OverviewPageProps> = ({onSubmit, modalLangua
 
   const atLeastOneChecked = Array.from(checked.values()).includes(true);
 
+  const mdComponents = {
+    img: ({...props}) => {
+      return (
+        <div className={styles.mdImgContainer}>
+          <img src={props.src} className={styles.mdImage}/>
+        </div>
+      );
+    },
+  };
 
   return (
     <div>
-      <div className = {wizardStyles.mainDivElement}>
-        <div className= {styles.center}>
-          <div className = {styles.comingSoon}>
-                Coming Soon!
-            <div className = "body3">
-              Mobile Preview is coming soon! Please double check all screens before submitting
+      <div className = {wizardStyles.mainDivElement}> 
+        <div className={styles.pageContainer}>
```

Comment:
There's also a lot of duplication here, let's make another component (in this file) like `<TabContentViewer content={read}>` to reduce the duplicate code.